Some changes in text + images (#1)
bakhtin/iroha-sandbox: iroha-transfer-asset/step1.md

This Iroha is pre-configured with two users: `alice@test` and `bob@test`. 

![Alice and Bob](assets/alicebob.png)

They both have [all available permissions](https://github.com/hyperledger/iroha/blob/master/shared_model/schema/primitive.proto#L29) in the system. 
So both of them can perform any of the available actions in the system. 

There is also an asset -- `coin#test`:

![Coin](assets/coin.png)

Alice, Bob and the asset are parts of the same domain - `test`. 
There might be other ones in the system, representing different households, companies or institutions.

> There is a [Python Iroha library](https://github.com/hyperledger/iroha-python/) installed in the system, so Iroha could process commands written within .py file. 

Reviewer: Iroha itself does not process commands written in Python. We interact with Iroha using Python library.

This Iroha is pre-configured with two users: `alice@test` and `bob@test`. 

![Alice and Bob](assets/alicebob.png)

They both have [all available permissions](https://github.com/hyperledger/iroha/blob/master/shared_model/schema/primitive.proto#L29) in the system. 
So both of them can perform any of the available actions in the system. 

There is also an asset -- `coin#test`:

![Coin](assets/coin.png)

Alice, Bob and the asset are parts of the same domain - `test`. 
There might be other ones in the system, representing different households, companies or institutions.

> There is a [Python Iroha library](https://github.com/hyperledger/iroha-python/) installed in the system, so commands written within .py file could be processed.

Reviewer: It is still a bit confusing. It is a Python library that helps us to interact with Iroha. `.py` files can be processed without the library itself. Can we rephrase to something like:
```suggestion
There is a [Python Iroha library](https://github.com/hyperledger/iroha-python/) installed in the system, so we can interact with Iroha from within a Python code
```
